scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -76,7 +76,7 @@ import kotlinx.coroutines.withContext
  */
 abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanner(name, config), CommandLineTool {
     companion object {
-        val DEFAULT_ARCHIVE_DIR by lazy { getOrtDataDirectory().resolve("scanner/archive") }
+        val DEFAULT_ARCHIVE_DIR by lazy { getOrtDataDirectory().resolve("archive") }
```

Comment:
This is a potentially breaking change for users that already have a scanner archive and should be mentioned in the commit message.